### PR TITLE
[New Rule] Persistence via Microsoft Outlook VBA

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/610

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/persistence_ms_outlook_vba_template.toml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+name = "Persistence via Microsoft Outlook VBA"
+references = [
+    "https://www.mdsec.co.uk/2020/11/a-fresh-outlook-on-mail-based-persistence/",
+    "https://www.linkedin.com/pulse/outlook-backdoor-using-vba-samir-b-/",
+]
+risk_score = 43
+rule_id = "397945f3-d39a-4e6f-8bcb-9656c2031438"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+type = "eql"
```
Comment (brokensound77) [inline rules/windows/persistence_ms_outlook_vba_template.toml:23]: Was the main reason for eql here to not have to escape the paths when using KQL wildcarding?

File: rules/windows/persistence_ms_outlook_vba_template.toml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+query = '''
+file where event.type != "deletion" and
+ wildcard(file.path, "C:\\Users\\*\\AppData\\Roaming\\Microsoft\\Outlook\\VbaProject.OTM") and
+ not wildcard(process.executable, "C:\\Program Files*\\Microsoft Office\\root\\Office*\\OUTLOOK.EXE")
```
Comment (brokensound77) [inline rules/windows/persistence_ms_outlook_vba_template.toml:28]: Curious on your thoughts to leaving this line out - which would detect if an attacker created the VBA from within outlook on the system. OTM files _should_ be rare enough that FP would be low - and you could add a snippet to `false_positives`.

File: rules/windows/persistence_ms_outlook_vba_template.toml (hunk 1 above)
Comment (Samirbous) [reply rules/windows/persistence_ms_outlook_vba_template.toml:23]: wildcard and case insensitivity support in the future with wildcard

File: rules/windows/persistence_ms_outlook_vba_template.toml (hunk 2 above)
Comment (Samirbous) [reply rules/windows/persistence_ms_outlook_vba_template.toml:28]: yep make sense (although doing it from within Outlook require interactive access to the machine like RDP)

**Review by dstepanic17 — APPROVED** [review]
This is great, LGTM. LinkedIn reference is 🔥

File: rules/windows/persistence_ms_outlook_vba_template.toml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/11/23"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/23"
+
+[rule]
+author = ["Elastic"]
+description = "Detects attempts to establish persistence on an endpoint by installing a rogue Microsoft Outlook VBA Template."
+false_positives = ["Legit VBA for Outlook usually configured interactively via OUTLOOK.EXE."]
```
Comment (bm11100) [inline rules/windows/persistence_ms_outlook_vba_template.toml:10]:
```suggestion
false_positives = ["A legitimate VBA for Outlook is usually configured interactively via OUTLOOK.EXE."]
```

File: rules/windows/persistence_ms_outlook_vba_template.toml [hunk 4]
```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/11/23"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/23"
+
+[rule]
+author = ["Elastic"]
+description = "Detects attempts to establish persistence on an endpoint by installing a rogue Microsoft Outlook VBA Template."
```
Comment (bm11100) [inline rules/windows/persistence_ms_outlook_vba_template.toml:9]: Maybe explain what VBA stands for and what it's used for to put it in context. I think that's optional, so going to approve regardless.

**Review by bm11100 — APPROVED** [review]

File: rules/windows/persistence_ms_outlook_vba_template.toml (hunk 4 above)
Comment (Samirbous) [reply rules/windows/persistence_ms_outlook_vba_template.toml:9]: details are in the 2 reference blogs

**Pull request merged**